lib/Constants.js

```diff
@@ -141,6 +141,7 @@ module.exports = {
       metadata.publish_date = data.microformat.playerMicroformatRenderer.publishDate || 'N/A';
       metadata.upload_date = data.microformat.playerMicroformatRenderer.uploadDate || 'N/A';
       metadata.keywords = data.videoDetails.keywords || [];
+      metadata.available_qualities = [...new Set(data[2].playerResponse.streamingData.adaptiveFormats.filter(v => v.qualityLabel).map(v => v.qualityLabel).sort((a, b) => +a.replace(/\D/gi, "") - +b.replace(/\D/gi, "")))]
```

Comment:
Amazing contribution!
Just one thing; the Innertube API sends a slightly different response for the desktop version of YouTube— so change this line right here to ``data.streamingData.adaptiveFormats`` and it should be good.
Thanks!

Reply:
Thank you, the change has been made.